setup.py

```diff
@@ -82,14 +82,15 @@ def determineCompiler(candidates, std, flags):
 		}
 	}""")
 	sample.close()
+	print(candidates)
 	for compiler in candidates:
-		cmd = [compiler,"-o","test_build","-std={}".format(std)]
+		cmd = [compiler,"-o","test_build","-std{}".format(std)]
 		cmd.extend(flags)
 		cmd.append("sample.cpp")
 		try:
 			if subprocess.call(cmd,stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL) == 0:
 				os.remove("sample.cpp")
-				os.remove("test_build")
+				os.remove("test_build*")
```

Comment:
Why is that change done? Does it even work? I don't think `os.remove` expands shell wildcards.

Reply:
It does work, but this is also a left-over from previous merge.